controller/lib/core/sensors.cpp

```diff
@@ -85,10 +84,24 @@ static int get_raw_sensor_zero_reading(AnalogPinId pinId) {
   return runningSum / zeroingAvgSize;
 }
 
-void zero_sensors() {
-  blower_disable();
-  Hal.delay(100); // some arbitrary time to wait for the pressure of the system
-                  // to equalize at all points
+void sensors_init() {
+  // Must wait at least 20 ms from power-on-reset for pressure sensors to
+  // warm-up (TODO: according to whom?).
+  //
+  // TODO: This sleep is unsafe!
+  //
+  //  - We can't safely get sensors' zero levels until the physical system is
+  //    quiescent.  In the event of a controller restart, 20ms doesn't seem
+  //    long enough; even it if is, we'd need to prove it.
+  //
+  //  - On the other hand, we can't wait a lot *longer* than 20ms because the
+  //    controller and GUI both have watchdogs that will restart the controller
+  //    if it hangs, and this sleep looks like a hang.
+  //
+  // It seems that we'll need to save calibration readings to non-volatile
+  // memory and provide operators with a way to shut off the device's blowers,
+  // open any necessary valves, and recalibrate.
+  Hal.delay(20);
```

Comment:
The 20 ms is the minimum time for the sensor to produce near accurate results after powering on. Straight from the datasheet. 

It is not the time it takes for the system to reach equilibrium from an air pressure perspective. This is unknown and would be estimated with actual experimentation. 

I don't think this initial sleep is unsafe. It is called once on POR. It can be done easily within the WDT period before the PID needs to do its thing. 

The comment is innacurate stating that the GUI can reset the controller; this is not possible in the alpha build. We rely on the WDT on the controller to do that. There is no physical way for the GUI computer to reset the controller, nor should there be.